posts/2023-06-20-introducing-leadership-council.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Introducing the Rust Leadership Council"
+author: Leadership Council
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+
+As of today, [RFC 3392] has been merged, forming the new top level governance body of the Rust Project: the Leadership Council. The creation of this Council marks the end of both the Core Team and the interim Leadership Chat.
+
+The Council will assume responsibility for top-level governance concerns while most of the responsibilities of the Rust Project (such as maintenance of the compiler and core tooling, evolution of the language and standard libraries, administration of infrastructure, etc.) remain with the nine top level teams.
+
+Each of these top level teams has chosen a representative who collectively form [the Council](https://www.rust-lang.org/governance/teams/leadership-council):
+- Compiler: Eric Holk
+- Crates.io: Carol (Nichols || Goulding)
+- Dev Tools: Eric Huss
+- Infrastructure: Ryan Levick
+- Language: Jack Huey
+- Launching Pad: Jonathan Pallant
+- Library: Mara Bos
+- Moderation: Khionu Sybiern
+- Release: Mark Rousskov
+
+First, we want to take a moment to thank the Core Team and interim Leadership Chat for the hard work they've put in over the years. Their efforts have been critical for the Rust Project. However, we do recognize that the governance of the Rust project has had its shortcomings. We hope to build on the successes and improve upon the failures to ultimately lead to greater transparency and accountability.
```

Comment:
nit but probably worth maintaining the capitalization for Project here too for consistency with the rest of the post

```suggestion
First, we want to take a moment to thank the Core Team and interim Leadership Chat for the hard work they've put in over the years. Their efforts have been critical for the Rust Project. However, we do recognize that the governance of the Rust Project has had its shortcomings. We hope to build on the successes and improve upon the failures to ultimately lead to greater transparency and accountability.
```